tests/assets/validator_test_data/stats.py

```diff
@@ -0,0 +1,6231 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
Can we reduce the size of testing code somehow? I wouldn't like to have a piece of write-only code.

Reply:
This is where the expected statistics/results of 'compute_statistics' method of the validator is stored (which is a rather long dictionary holding information). Are you suggesting that I should reduce the size of the test dataset (in 'tests/assets/validator_test_data/datasets.py') that corresponds with the expected stats (which would reduce the size of the expected stats)? Or are you suggesting that I should cut down the main test script in 'tests/test_validator.py'? Perhaps both?

Reply:
I suggest to think about maintaining these tests. If the answer to the questions "is it easy to maintain them? Would you like to do it on a constant basis?" is "no", then, we're probably moving in a wrong direction. I can recommend few ways to improve:
- test only what is necessary in "common", function-specific tests, and have 1-2 more complex, general tests
- do not test full text of error messages (unless it is a direct business requirement)
- minimize test scenarios, deduplicate (eg. leave only single subset instead of multiple, leave only single `DatasetItem`, if it is enough, leave only single `Bbox`, if it is enough etc.)

Reply:
Got it! I got rid of tests/assets/validator_test_data entirely and reworked the unit test along with other changes according to your comments on the most recent commit.